exercises/concept/errors/package.json

```diff
@@ -1,36 +1,31 @@
 {
   "name": "@exercism/javascript-concept-errors",
   "author": "Tom Pradat <tom.pradat@gmail.com>",
-  "version": "1.0.0",
-  "license": "MIT",
   "private": true,
   "repository": {
     "type": "git",
-    "url": "https://github.com/exercism/language-concepts.git",
+    "url": "https://github.com/exercism/javascript",
     "directory": "languages/javascript/exercises/concept/errors"
```

Comment:
This should be updated. I have created an issue to automatically set this. https://github.com/exercism/javascript/issues/1007

Reply:
I've apparently missed this comment, this is done in https://github.com/exercism/javascript/pull/978/commits/6d0f62026ccbb6f7f7e15e92bd4223018b7babab :ok_hand: